server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -118,11 +144,15 @@ void beforeEach() throws Exception {
         JsonTransformer jsonTransformer = new JsonTransformer();
 
         RestoreService vaultRestore = new RestoreService(vault, mailboxManager);
+        blobSharing = spy(new DummyBlobSharing());
+        blobStore = spy(new MemoryBlobStore(new HashBlobId.Factory()));
+        zipper = spy(new DeletedMessageZipper());
```

Comment:
Why do we need to spy this????

Please asserts the results (spying a class dedicated to side effect is OK) but by spying the Zipper you are for sure asserting the `how`